src/tests/inventorySorting.spec.js

```diff
@@ -0,0 +1,69 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+
+const cleanPrices = (prices) => prices.map(price => parseFloat(String(price).replace(/$/g, '')));
+
+const sortPrices = (prices, order) => {
+    const cleanedPrices = cleanPrices(prices);
+    return order === 'asc' ? cleanedPrices.sort((a, b) => a - b) : cleanedPrices.sort((a, b) => b - a);
+};
+
```

Comment:
It is a great idea to create those. Usually, we are using a separate file to store the helper functions like that